recipes/sample-sheet/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+$PYTHON setup.py install --single-version-externally-managed --record=record.txt
```

Comment:
Not critical, but since this is a one-line build, you could get rid of the file and instead add `script: $PYTHON...` to the `build:` section in the `meta.yaml`.